api/src/services/email.service.ts

```diff
@@ -533,24 +533,24 @@ export class EmailService {
       );
       tableRows.push({
         label: this.polyglot.t(`rentalOpportunity.${key}`),
-        value: `${bedroom.length} unit${
+        value: `${bedroom?.length} unit${
```

Comment:
How does this display if bedroom is null (`undefined unit`?)?

Reply:
good question. It would probably display `undefined unit`. This shouldn't ever happen, I was just testing with a bare minimum listing (in a flow that would never happen) and I ran into several null pointers on this page. I just put a bunch of checks to prevent the thing from totally failing.